front/components/assistant/conversation/AgentMessage.tsx

```diff
@@ -139,6 +170,7 @@ export function AgentMessage({
     switch (message.status) {
       case "succeeded":
       case "failed":
+      case "cancelled":
```

Comment:
No particular UI for cancelled mesages? We should likely show something otherwise another viewer of the conversation could believe it's a bug no?

Reply:
Indeed for another user it could look like a bug. I'll ask @Duncid and implement something accordingly, in another PR today unless you think it's a blocker.

Reply:
Not a blocker

Reply:
https://dust4ai.slack.com/archives/C050A0S2Z7F/p1696924941531459